modules/apps/headless/headless-builder/headless-builder-test/src/testIntegration/java/com/liferay/headless/builder/application/publisher/test/APIApplicationPublisherTest.java

```diff
@@ -81,19 +81,6 @@ public Application addingService(
 
 	@After
 	public void tearDown() throws Exception {
-		HTTPTestUtil.invoke(
-			null,
-			"headless-builder/applications/by-external-reference-code/" +
-				_API_APPLICATION_ERC_1,
-			Http.Method.DELETE);
-		HTTPTestUtil.invoke(
-			null,
-			"headless-builder/applications/by-external-reference-code/" +
-				_API_APPLICATION_ERC_2,
-			Http.Method.DELETE);
-
-		_serviceTracker.close();
-
 		APIApplicationPublisherUtil.unpublishRemainingAPIApplications(
```

Comment:
I sent a PR to avoid using the tearDown method from the parent. If for some reason, the PR is merged, I will remove the new `super.tearDown()` invocation

Reply:
Done